### DISCUSSION TITLE
How to make verdaccio able to search all cached modules in the search box on the home page, not just user-published modules?

### DISCUSSION THREAD

**Discussion by senro** [verdaccio/verdaccio#3832] [Ideas]
**Is your feature request related to a problem?**
Please describe a clear and concise description of what the problem is. E.g. I'm always frustrated when [...]

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by juanpicado** [comment]
v6 already does that, but is not perfect yet. 

```
npm install --location=global verdaccio@6-next
```

**Comment by juanpicado** [comment]
With the latest 6.x looks like this


https://github.com/verdaccio/verdaccio/assets/558752/98703478-a104-42cc-82d3-e5a4f50f6d64
